frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -121,39 +225,341 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
+
+const OperandDeletionSuccess: React.FC<{ name: string; namespace: string }> = ({
+  name,
+  namespace,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <Alert
+      variant="success"
+      className="co-alert"
+      title={t('olm~Successfully deleted all Operand Instances')}
+      isInline
+    >
+      <p>
+        <Trans t={t} ns="olm">
+          All Operand instances for Operator <strong>{{ name }}</strong> in{' '}
+          <strong>{{ namespace }}</strong> have been deleted.
+        </Trans>
+      </p>
+    </Alert>
+  );
+};
+
+const UninstallAlert: React.FC<{ errorMessage: string; name: string; namespace: string }> = ({
+  errorMessage,
+  name,
+  namespace,
+}) =>
+  errorMessage ? (
+    <OperatorUninstallError errorMessage={errorMessage} />
+  ) : (
+    <OperatorUninstallSuccess name={name} namespace={namespace} />
+  );
+
+const OperandDeletionAlert: React.FC<{
+  errors: OperandWithErrorProps[];
+  csvName: string;
+  cancel: () => void;
+  name: string;
+  namespace: string;
+}> = ({ errors, csvName, cancel, name, namespace }) =>
+  errors.length ? (
+    <OperandDeletionErrors operandDeletionErrors={errors} csvName={csvName} cancel={cancel} />
+  ) : (
+    <OperandDeletionSuccess name={name} namespace={namespace} />
+  );
+
+const SubmitResults: React.FC<SubmitResultsProps> = ({
+  operatorUninstallErrorMessage,
+  name,
+  namespace,
+  isDeleteAllOperandsSelected,
+  operandDeletionErrors,
+  csvName,
+  cancel,
+}) => (
+  <>
+    <UninstallAlert
+      errorMessage={operatorUninstallErrorMessage}
+      name={name}
+      namespace={namespace}
+    />
+    {isDeleteAllOperandsSelected && (
+      <OperandDeletionAlert
+        csvName={csvName}
+        errors={operandDeletionErrors}
+        cancel={cancel}
+        name={name}
+        namespace={namespace}
+      />
+    )}
+  </>
+);
+
+export const OperandsTable: React.FC<OperandsTableProps> = ({
+  operands,
+  loaded,
+  csvName,
+  cancel,
+}) => {
+  const { t } = useTranslation();
+  return (
+    <StatusBox
+      skeleton={<div className="loading-skeleton--table" />}
+      data={operands}
+      loaded={loaded}
+    >
+      <table className="pf-c-table pf-m-compact pf-m-border-rows">
+        <thead>
+          <tr>
+            <th>{t('olm~Name')}</th>
+            <th>{t('olm~Kind')}</th>
+            <th>{t('olm~Namespace')}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {_.map(operands, (operand, i) => (
+            <>
+              <tr key={i}>
+                <td>
+                  <OperandLink obj={operand} csvName={csvName} cancel={cancel} />
+                </td>
+                <td className="co-break-word" data-test-operand-kind={operand.kind}>
+                  {operand.kind}
+                </td>
+                <td>
+                  {operand.metadata.namespace ? (
+                    <ResourceLink
+                      kind="Namespace"
+                      title={operand.metadata.namespace}
```

Comment:
No need for a `title` here unless the CSS styles could cause the namespace name to be truncated.